src/test/java/teammates/test/cases/webapi/UpdateFeedbackResponseActionTest.java

```diff
@@ -25,6 +27,18 @@
  */
 public class UpdateFeedbackResponseActionTest extends BaseActionTest<UpdateFeedbackResponseAction> {
 
+    private StudentAttributes student1InCourse1;
+    private StudentAttributes student2InCourse1;
+    private StudentAttributes student5InCourse1;
+    private InstructorAttributes instructor2OfCourse1;
+    private InstructorAttributes instructor1OfCourse1;
+    private FeedbackResponseAttributes typicalResponse;
+    private FeedbackResponseAttributes testModerateResponse;
+    private FeedbackResponseAttributes typicalResponse2;
+    private FeedbackResponseAttributes typicalResponse3;
+    private FeedbackResponseAttributes responseInClosedSession;
+    private FeedbackSessionAttributes closedSession;
```

Comment:
Let's try to reduce the amount of class variables used, where possible.
closedSession (and responseInClosedSession) is only used in one method (testAccessControl_responseSessionNotOpen_inaccessible), we can refactor all the code into there. 
Also let's have more descriptive names for the responses. Instead of typicalResponse, how about "student1ResponseToStudent1".